config/ocm-requests.json

```diff
@@ -0,0 +1,16 @@
+{
+  "elasticsearch": {
+    "ocm-requests": [
+      {
+        "filter": {
+          "test_name.keyword": "self-terms-review"
+        },
+        "buckets":[
+          "code",
+          "latency_ns"
+        ],
```

Comment:
This kind of query, "timeseries", does not use buckets. Remove them from here, pls